tests/README.md

```diff
@@ -0,0 +1,33 @@
+## Frontend Tests 
```

Comment:
nit: maybe this should be a larger heading than subheadings?
```
# Frontend Tests
```

or we make the sub headings smaller? what do you think?

Reply:
Good idea, Thank you.